cli/cluster/yaml.go

```diff
@@ -0,0 +1,35 @@
+package cluster
+
+import (
+	"fmt"
+
+	"gopkg.in/yaml.v2"
+)
+
+// YamlCollector collects a configuration from YAML data.
+type YamlCollector struct {
+	data []byte
+}
+
+// NewYamlCollector create a new YAML collector.
+func NewYamlCollector(data []byte) Collector {
+	return YamlCollector{
+		data: data,
+	}
+}
+
+// Collect collects a configuration from YAML data.
+func (c YamlCollector) Collect() (*Config, error) {
+	var t map[any]any
```

Comment:
Please, use meaningful names for variables.

Reply:
Replaced by `parser`.